src/app/features/report/report/components/data-drift/data-drift.component.html

```diff
@@ -1,24 +1,54 @@
 <div
-  class="summary"
-  [ngClass]="driftedFeatures.length > 0 ? 'summary--bad' : 'summary--excellent'"
+  class='summary'
 >
-  <div class="summary__title">Data drift</div>
-  <div class="summary__cell">
-    <hs-icon class="w-6 h-6 absolute top-2 right-2 text-gray-400" name="info" matTooltip="drifted features count / total features"></hs-icon>
-    <div
-      class="summary__status"
-    >
-      {{ driftedFeatures.length }}/{{featuresNumber}}
+  <ng-container *ngIf='driftedFeatures.length > 0; else ok'>
+    <svg xmlns='http://www.w3.org/2000/svg' class='h-6 w-6 absolute right-14 text-red-500' viewBox='0 0 20 20'
+         fill='currentColor'>
+      <path fill-rule='evenodd'
+            d='M10 18a8 8 0 100-16 8 8 0 000 16zM8.707 7.293a1 1 0 00-1.414 1.414L8.586 10l-1.293 1.293a1 1 0 101.414 1.414L10 11.414l1.293 1.293a1 1 0 001.414-1.414L11.414 10l1.293-1.293a1 1 0 00-1.414-1.414L10 8.586 8.707 7.293z'
+            clip-rule='evenodd' />
+    </svg>
```

Comment:
In order to maintain common style, I think it would be better to use icons from hs-ui-kit lib as it did now.